src/swap/server/volume_contest.py

```diff
@@ -0,0 +1,105 @@
+from decimal import Decimal
+from typing import Optional
+from dataclasses import dataclass, field
+from datetime import datetime, date, timezone, timedelta
+
+import strawberry
+from pymongo.database import Database
+from strawberry.types import Info
+
+
+STABLE_PAIRS = {
+    "0x5801bdad32f343035fb242e98d1e9371ae85bc1543962fedea16c59b35bd19b",
+    "0xcfd39f5244f7b617418c018204a8a9f9a7f72e71f0ef38f968eeb2a9ca302b",
+    "0xf0f5b3eed258344152e1f17baf84a2e1b621cd754b625bec169e8595aea767"
+}
+WEEKLY_HARD_CAP = Decimal(1000)
+STABLE_POOL_MULTIPLIER = Decimal(0.5)
+
+
+@dataclass
+class Week:
+    _id: int
+    start_dt: datetime
+    end_dt: datetime
+    volume: int = 0
```

Comment:
We need both volume and score for each week. Tagged you on discord, on the design.

Reply:
added all data related to Week:
      id
      name
      startDt
      endDt
      score
      volume